public/components/experiment_listing/experiment_listing.tsx

```diff
@@ -166,8 +168,16 @@ export const ExperimentListing: React.FC<ExperimentListingProps> = ({ http, hist
         hits: filteredList,
       };
     } catch (err) {
-      console.error('Failed to load experiment', err);
-      setError('Failed to load experiments');
+      console.error('Failed to load experiments', err);
+      if (err.body && err.body.message === DISABLED_BACKEND_PLUGIN_MESSAGE) {
```

Comment:
can this block of error handling be made more generic so we don't have the same code in multiple files?  Pretty much any api call is going to need this right?   Maybe wherever we actually send the call is the right place to check this, versus each component do it independently?

Reply:
I tried something different that would let us handle errors differently, and not have as much duplicate code in each file. Let me know if that goes in the right direction and I can update the other files.